.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+workflows:
+  version: 2
+  test:
+    jobs:
+      # TODO(karita): support tests without conda
+      - test-ubuntu18
+      - test-ubuntu16
+      - test-centos7
+
+jobs:
+  test-ubuntu18:
+    docker:
+      - image: circleci/buildpack-deps:18.04
+        environment:
+          ESPNET_PYTHON_VERSION: 3.7
+          TH_VERSION: 1.1.0
+          CHAINER_VERSION: 5.0.0
+          USE_CONDA: true
+          CC: gcc-7
+          CXX: g++-7
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run:
+          name: run tests
+          command: |
+            sudo apt-get install -qq -y cmake g++-7 libsndfile1-dev
+            ./ci/install.sh
+            ./ci/test_shell.sh
+            ./ci/test_python.sh
+
+  test-ubuntu16:
+    docker:
+      - image: circleci/buildpack-deps:16.04
+        environment:
+          ESPNET_PYTHON_VERSION: 3.7
+          TH_VERSION: 1.1.0
+          CHAINER_VERSION: 5.0.0
+          USE_CONDA: true
+          CC: gcc-7
+          CXX: g++-7
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - run:
+          name: run tests
+          command: |
+            sudo apt-get install -y software-properties-common
+            sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+            sudo apt-get update -qq
+            sudo apt-get install -qq -y cmake g++-7 libsndfile1-dev
+            ./ci/install.sh
+            ./ci/test_shell.sh
+            ./ci/test_python.sh
+
+  test-centos7:
+    docker:
+      - image: centos:7
+        environment:
+          ESPNET_PYTHON_VERSION: 3.7
+          TH_VERSION: 1.1.0
+          CHAINER_VERSION: 5.0.0
+          USE_CONDA: true
+          CC: /opt/rh/devtoolset-7/root/usr/bin/gcc
+          CXX: /opt/rh/devtoolset-7/root/usr/bin/g++
+    working_directory: ~/repo
+
+    steps:
+      - run:
+          name: Install dependencies
+          command: |
+            yum install -y git centos-release-scl cmake libsndfile make bzip2 wget which unzip
+            yum-config-manager --enable rhel-server-rhscl-7-rpms
+            yum install -y devtoolset-7-gcc-c++
+      - checkout
+      - run:
+          name: run tests
+          command: |
+            # NOTE(karita) this line exited 1
+            # source scl_source enable devtoolset-7
+            PATH="/opt/rh/devtoolset-7/root/usr/bin:${PATH:-}"
```

Comment:
This line is used instead of  `source scl_source enable devtoolset-7` ? 
(I want to just confirm)

Reply:
Yes. This is very strange. When I ssh into circle ci, it accepts `source scl_source enable devtoolset-7` but `config.yml` does not.

Reply:
:(

Reply:
Anyone can retry that by "Rerun workflow > Rerun job with ssh" https://circleci.com/gh/espnet/espnet/59